pkg/dao/repository_config_test.go

```diff
@@ -1027,7 +1027,7 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 	assert.True(t, response.URL.MetadataPresent)
 	assert.False(t, response.URL.Skipped)
 
-	mockExtRDao.Mock.On("ValidRepoMD", "http://badrepo.com").Return(404, nil)
+	mockExtRDao.Mock.On("ValidRepoMD", "http://badrepo.com/").Return(404, nil)
```

Comment:
see comments above, can you use
```suggestion
	mockExtRDao.Mock.On("ValidRepoMD", "http://badrepo.test/").Return(404, nil)
```